src/components/select/Select.js

```diff
@@ -855,13 +855,16 @@ export default class SelectComponent extends Field {
       searchFields: _.get(this, 'component.searchFields', ['label']),
       shadowRoot: this.root ? this.root.shadowRoot : null,
       fuseOptions: this.component.useExactSearch
-        ? {}
+        ? {
+          tokenize: true,
+          matchAllTokens: true,
+        }
         : Object.assign(
         {},
         _.get(this, 'component.fuseOptions', {}),
         {
           include: 'score',
-          threshold: _.get(this, 'component.searchThreshold', 0.3),
```

Comment:
So, the configuration has always been "selectThreshold" but we use "searchThreshold" in the code?

Reply:
yes, that is true, the key of the 'search threshold' option in our builder has always been "selectThreshold", but in the code we used "searchThreshold" instead and this option did not work.